tests/ignite/engine/test_create_supervised.py

```diff
@@ -105,12 +119,59 @@ def test_create_supervised_trainer_traced_with_cpu():
     _test_create_supervised_trainer(trainer_device="cpu", trace=True)
 
 
+def test_create_supervised_trainer_apex_error():
+    with pytest.raises(
+        ModuleNotFoundError, match="Please install apex from https://github.com/nvidia/apex to use amp_mode='apex'."
+    ):
+        _test_create_supervised_trainer(amp_mode="apex")
+
+
+@pytest.mark.skipif(LooseVersion(torch.__version__) > LooseVersion("1.6.0"), reason="Skip if > 1.6.0.")
+def test_create_supervised_trainer_amp_error():
```

Comment:
Can we make this test work in torch > 1.6?
I don't know how to make it work if torch > 1.6

Reply:
`amp` is available since PyTorch 1.6 so it can't have an error raised with torch > 1.6.

I don't understand what you mean...

Reply:
That test is for checking `cuda.amp` can be imported or not.
Since tests are running with latest pytorch version, this test is skipped in the unit tests.
And I want to make this test run for coverage

Reply:
You can use mock.patch with a context manager to temporarily patch `cuda.amp` module and thus be able to raise the tested error. Take a look here for example: https://github.com/pytorch/ignite/blob/master/tests/ignite/contrib/metrics/test_gpu_info.py#L91